modules/tinymce/CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The formatter can now apply a format to a `contenteditable="false"` element by wrapping it. Configurable using the `format_noneditable_selector` option. #TINY-8905
 - The autocompleter now supports a multiple character trigger using the new `trigger` configuration #TINY-8887
 
+### Changed
+- `@menubar-row-separator-color` variable now has no effect on the divider between Menubar and Toolbar and only controls the color of the separator lines drawn in multiline Menubar. `@toolbar-separator-color` variable is now exclusively used affects the color of separator between Menubar and Toolbar #TINY-8632
```

Comment:
```suggestion
### Changed
- The `@menubar-row-separator-color` variable no longer affects the divider between the Menubar and Toolbar and only controls the color of the separator lines drawn in multiline Menubars. #TINY-8632
- The `@toolbar-separator-color` variable now exclusively affects the color of the separator between the Menubar and Toolbar. #TINY-8632
```
This is 2 separate things and is too long for a changelog anyways, so splitting it makes sense to me. This also still didn't really make sense so I've tried to reword it a little.

Reply:
@noxuhax why was this marked as resolved without making any changes or leaving any comments?

Reply:
@lnewson my bad, I did not get notifications about new comments so I was convinced that I’m marking an old thread as resolved. I agree, splitting this entry into two makes more sense.